packages/i18n/Uppercase/Uppercase.js

```diff
@@ -25,6 +26,8 @@ const formatContent = (casing, content) => {
 	}
 };
 
+deprecate({name: 'i18n/Uppercase', until: '3.0.0', replacedBy: 'i18n/util', since: '2.6.0'});
```

Comment:
This is questionable value but it'd be "nice" to only warn when authors use `Uppercase` but not the framework. Previously, we've accomplished this by exporting a `private` version unwrapped by `deprecate` use it within the framework.

```
const WrappedUppercase = deprecate( ... );
export default WrappedUppercase;
export {
  WrappedUppercase as Uppercase,
  Uppercase as privateUppercase
};
```

I'm having trouble justifying the extra work but something to consider.

Reply:
Done!